src/components/shared/buttons/ShareButton.tsx

```diff
@@ -27,5 +27,14 @@ export const ShareButton: React.FC<Props> = ({ title = '', text = '' }) => {
     }
   }
 
-  return <Button onClick={handleShare} icon={Share2} iconOnly variant='secondary' />
+  return (
+    <Button
+      onClick={handleShare}
+      size={'sm'}
+      icon={Share2}
+      iconOnly
+      variant='secondary'
+      className='h-[42px] rounded-[10px]'
```

Comment:
I'm repeating myself, there is no need to override class height and border.
We should use size='md' here and that's it, no need to override className

You can check that size='md' have the same specifications that you've sent within `className`.